apps/src/music/player/sequencer/AdvancedSequencer.ts

```diff
@@ -124,6 +124,6 @@ export default class AdvancedSequencer extends Sequencer {
   }
 
   getLastMeasure(): number {
-    return 0;
+    return 100;
```

Comment:
Could we compute this whenever we add an event by checking the max of current last measure and newEvent.when + newEvent.length? Fine to leave as is for now though since it seems to be mostly a placeholder.